content/php-client/resources.md

```diff
@@ -0,0 +1 @@
+# Resources
```

Comment:
We already use this term in the general documentation. That's ok, from my point of view.